aptos-move/block-executor/src/executor.rs

```diff
@@ -137,61 +142,54 @@ where
             .modified_keys(idx_to_execute)
             .map_or(HashMap::new(), |keys| keys.collect());
 
-        let mut prev_modified_aggregators = last_input_output
-            .aggregator_v2_keys(idx_to_execute)
-            .map_or(HashSet::new(), |keys| keys.collect());
+        // let mut prev_modified_aggregators = last_input_output
```

Comment:
If we don't validate the reads during normal validation (which we don't atm), then this clearly isn't adding anything.
Even if we do the optimized flow, it'd potentially allow finding affected transactions but the trade-off is pretty unclear (causing a whole new wave). We could design some mechanism in the future, but for now I'd get rid of it. The key is that we sequentially make sure the V2 behavior checks out, so don't need BlockSTM to be correct for it. (maybe we can add this comment to why we are ignoring the delta locations).
@sasha8 to confirm my reasoning here.